styleguide/STYLE.md

```diff
@@ -192,6 +192,20 @@ Internal links need to start with `/docs`, for example:
 Read more about [environment variables](/docs/pipelines/environment-variables)
 ```
 
+### Content reuse
+
+Add snippets to the directory where they'll be used, prefaced with an underscore. For example `_my_snippet.md.erb`. **However**, when pulling the snippet into a file, remove the leading underscore.
+
+Thus
```

Comment:
can we avoid thus, even in the style guide?

Reply:
thus -> so